mlflow/pyspark/ml/__init__.py

```diff
@@ -40,12 +41,19 @@
 
 def _read_log_model_allowlist_from_file(allowlist_file):
     allowlist = set()
-    with open(allowlist_file) as f:
-        for line in f:
-            stripped = line.strip()
-            is_blankline_or_comment = stripped == "" or stripped.startswith("#")
-            if not is_blankline_or_comment:
-                allowlist.add(stripped)
+    try:
+        response = requests.get(allowlist_file)
```

Comment:
Can we use the utility method `http_request` https://github.com/mlflow/mlflow/blob/d382b2dd92065a68ea7ab6db16980fc65e1a8dca/mlflow/store/artifact/http_artifact_repo.py#L63 ? 
It includes retry setting.